homework/06LessonSix.js

```diff
@@ -20,22 +22,27 @@ var point = {
 // In order to do this, you should add parameters x and y
 // as properties to "this".
 // example: this.propName = propValue;
-function Point(x, y) {
-}
+function Point(paramX, paramY) {
+  this.x = paramX;
+  this.y = paramY;
+};
 
 // Problem 3:
 // Create a new point using the class constructor Point.
 // Set x to 5, and y to -3
 // This can be done as follows: new Point(someX, someY)
-var anotherPoint;
+var anotherPoint = new Point( );
```

Comment:
We can improve this. `Point(paramX, paramY)` has params that let us automatically set the x and y properties. Let's use them!